truffle/src/main/java/org/jruby/truffle/runtime/RubyContext.java

```diff
@@ -309,6 +311,10 @@ public RubyString makeString(RubyClass stringClass, char string, Encoding encodi
         return makeString(stringClass, Character.toString(string), encoding);
     }
 
+    public RubyString makeString(byte[] bytes) {
```

Comment:
This would make them inaccessible from non-Node callers, which is undesirable though.